src/elements/content-sidebar/SidebarNav.js

```diff
@@ -61,6 +61,7 @@ const SidebarNav = ({
                         data-testid="sidebaractivity"
                         sidebarView={SIDEBAR_VIEW_ACTIVITY}
                         tooltip={<FormattedMessage {...messages.sidebarActivityTitle} />}
```

Comment:
Should we have the SidebarNavButton component apply the `aria-label` attribute, instead? It would save us from having to pass it twice.

Reply:
@jstoffan We should add `aria-label` attribute directly in HTML Elements. In this case we should pass that attribute until it is applied to an HTML element. I found an example where they do something similar https://github.com/box/box-ui-elements/blob/85c005a4de9cae201d3a879d08277ac9eafe5b89/src/components/grid-view/GridViewSlider.js#L38

Reply:
I'm sorry, I think we're talking about different things. I'm suggesting that we reuse the localized string for tooltip by changing this line to:

```
tooltip={formatMessage(messages.sidebarActivityTitle)}
```

Then, in SidebarNavButton.js, we can add:

```
<Tooltip position="middle-left" text={tooltip} isTabbable={false}>
    <NavButton
        aria-label={tooltip}
        // ...other props ...
```

What do you think?

Reply:
@jstoffan sorry I misunderstood what are saying in the previous comment. I agree with you. I will update my PR. Thank you!

Reply:
No problem, thanks for taking a look!